src/components/ProfileCard/ProfileCard.tsx

```diff
@@ -0,0 +1,170 @@
+
+  const formatRanking = (rank: number): string => {
+    if (rank === 0) return '-';
+
+    const lastDigit = rank % 10;
+    const lastTwoDigits = rank % 100;
+
+    // Special cases for 11, 12, 13 (always 'th')
+    if (lastTwoDigits >= 11 && lastTwoDigits <= 13) {
+      return `${rank}th`;
+    }
+
+    // Regular cases based on last digit
+    if (lastDigit === 1) return `${rank}st`;
+    if (lastDigit === 2) return `${rank}nd`;
+    if (lastDigit === 3) return `${rank}rd`;
+
+    return `${rank}th`;
+  };
+
+  return (
+    <StyledCard
+      className={className}
+      data-testid={dataTestId}
+      $size={size}
+      $variant={variant}
+      role="article"
+      aria-label={`Profile card for ${name}`}
+      {...cardProps}
+    >
+      <StyledProfileContainer $size={size} $variant={variant}>
+        <StyledAvatar
+          src={avatarUrl}
+          alt={`${name}'s avatar`}
+          $size={size}
+          data-testid={`${dataTestId}-avatar`}
+        >
+          {!avatarUrl && initials && initials}
+          {!avatarUrl && !initials && <DefaultIcon />}
+        </StyledAvatar>
+
+        <StyledInfoContainer $size={size} $variant={variant}>
+          <StyledName variant="h6" $size={size} title={name} data-testid={`${dataTestId}-name`}>
+            {name}
+          </StyledName>
+          <StyledUsername
+            variant="body2"
+            $size={size}
+            title={username}
+            data-testid={`${dataTestId}-username`}
+          >
+            @{username}
+          </StyledUsername>
+          {role && (
+            <StyledRole
+              variant="body2"
+              $size={size}
+              title={role}
+              data-testid={`${dataTestId}-role`}
+            >
+              {role}
+            </StyledRole>
+          )}
+        </StyledInfoContainer>
+      </StyledProfileContainer>
+
+      {showStats && (
+        <StyledStatsContainer
+          $size={size}
+          $variant={variant}
+          data-testid={`${dataTestId}-stats`}
+          role="region"
+          aria-label="User statistics"
+        >
+          <StatsItem
+            icon={<STATS_ICONS.courses />}
+            label={STATS_LABELS.courses}
+            value={stats.courses}
+            $variant={variant}
+            data-testid={`${dataTestId}-stat-courses`}
+          />
+          <StatsItem
+            icon={<STATS_ICONS.points />}
+            label={STATS_LABELS.points}
+            value={stats.points}
+            $variant={variant}
+            data-testid={`${dataTestId}-stat-points`}
+          />
+          <StatsItem
+            icon={<STATS_ICONS.ranking />}
+            label={STATS_LABELS.ranking}
+            value={formatRanking(stats.ranking)}
+            $variant={variant}
+            data-testid={`${dataTestId}-stat-ranking`}
+          />
+        </StyledStatsContainer>
+      )}
+    </StyledCard>
+  );
+};
+
+export default ProfileCard;
```

Comment:
This component file ends with a default export `export default ProfileCard;`. The repository conventions mandate named exports only. Keep `export const ProfileCard` and remove the default export. If Storybook or other tooling requires a default export somewhere, scope that exception only to those tooling files and document it.

**Sugerencia:**
```typescript
Remove the default export and rely on the named export only. Update any imports that relied on the default export to use the named import: `import { ProfileCard } from './ProfileCard'`.
```